techniques/general/G180.html

```diff
@@ -51,74 +49,73 @@
+<section id="description"><h2>解説</h2>
+      <p>このテクニックの目的は、障害のない利用者に比べて障害のある利用者が長時間必要とするかもしれないタスクにおいて、障害のある利用者に完了するための十分な時間を提供することである。環境設定又はページ上のコントロールのようなメカニズムによって、利用者がデフォルトの制限時間の少なくとも 10 倍まで制限時間を延長できるようにする。可能であれば、このメカニズムはさまざまな調整機能を備え、利用者が制限時間を範囲内の任意の値に変えられるだけでなく、任意の増減幅で変えられる方法も提供するとよい。これにより、利用者は、制限時間のあるセッションを始める前に、制限時間を変更できるようになる。</p>
+   </section><section id="examples"><h2>事例</h2>
       <ul>
-         <li>An airline has an online ticket purchasing application. By default, the application has a 1 minute time limit for each step of the purchase process. At the beginning of the session, a web page includes information that says, "We expect that each step in the purchasing process will take users one minute to complete. Would you like to adjust the time limit?" followed by several radio buttons "1 minute, 2 minutes, 5 minutes, 10 minutes."</li>
-         <li>A Web based email application automatically logs users out when there has been no activity for 30 minutes. The application includes a preference that allows users to adjust the amount of time to any value.</li>
+         <li>航空会社が、オンラインで航空券を購入するためのアプリケーションを提供している。このアプリケーションは、購入プロセスの各ステップに対して 1 分間の制限時間をデフォルトで設定している。セッションの開始時に、「購入プロセスの各ステップには制限時間があり、1分以内に完了させる必要があります。この制限時間を調整しますか？」とウェブページに表示され、続いて「1 分」「2 分」「5 分」「10 分」などのラジオボタンがいくつか表示されている。</li>
+         <li>ウェブベースの E メールアプリケーションで、30 分間利用者の操作がないと、自動的にログアウトするようになっている。このアプリケーションには、利用者がタイムアウト時間を任意の値に調整できる環境設定が含まれている。</li>
       </ul>
    </section>
-<section id="related"><h2>Related Techniques</h2><ul>
+<section id="related"><h2>関連テクニック</h2><ul>
       <li><a href="../general/G133">G133: Providing a checkbox on the first page of a multipart form that allows users to ask for longer session time limit or no session time limit</a></li>
       <li><a href="../client-side-script/SCR1">SCR1: Allowing the user to extend the default time limit</a></li>
       <li><a href="../client-side-script/SCR16">SCR16: Providing a script that warns the user a time limit is about to expire</a></li>
    </ul></section>
-<section id="tests"><h2>Tests</h2>
-      <section class="procedure" id="procedure"><h3>Procedure</h3>
+<section id="tests"><h2>検証</h2>
+      <section class="procedure" id="procedure"><h3>手順</h3>
          <ol>
-            <li>Check to see if there is a mechanism to set the time limit to 10 times the default time limit.</li>
-            <li>Change the time limit to a new value that is 10 times the default time limit.</li>
-            <li>Perform an action that has a time limit.</li>
-            <li>Wait until the default time limit has passed.</li>
-            <li>Check that the time limit does not expire until the limit specified in step 2 has passed.</li>
+            <li>制限時間をデフォルトの 10 倍に設定するメカニズムがあるかどうかを確認する。</li>
+            <li>制限時間をデフォルトの 10 倍にあたる新しい値に変更する。</li>
+            <li>制限時間のあるタスクを遂行する。</li>
+            <li>デフォルトの制限時間が経過するのを待つ。</li>
+            <li>上記 2 で設定した制限時間になるまで時間切れにならない。</li>
```

Comment:
check that ... を訳出しました。
```suggestion
            <li>上記 2 で設定した制限時間になるまで時間切れにならないことをチェックする。</li>
```